src/__tests__/Layout.test.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * @format
+ */
+
+import React from 'react'
+import 'react-native'
+import { Text } from 'react-native'
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+import { Layout } from '../components/Layout'
+
+it('renders the layout', () => {
```

Comment:
Not a very solid test. This will be addressed in a new issue so that all components
get tested well.